Buddies.Web/src/components/Header.tsx

```diff
@@ -72,7 +73,9 @@ const Header = ({
         </Grid>
         <Grid container item xs={2} justifyContent="flex-end">
 
-          <CopyToClipboard text="www.testurl.com">
+          <CopyToClipboard
+            text={`http://buddy-alb-1653424214.us-east-1.elb.amazonaws.com/profiles/${pageId}`}
```

Comment:
Generally you want to either inject the URL, or you can get the base URL of the current page [using JS](https://www.w3schools.com/js/js_window_location.asp)